AuthSamples/Sample/MainViewController.m

```diff
@@ -539,10 +535,6 @@ - (id)initWithNibName:(nullable NSString *)nibNameOrNil bundle:(nullable NSBundl
                                                  name:FIRAuthStateDidChangeNotification
                                                object:nil];
     self.useStatusBarSpinner = YES;
-#if INTERNAL_GOOGLE3_BUILD
-    // Trigger automatic token refresh.
-    [[FIRDatabase database] reference];
```

Comment:
Not automatically, but it can still be triggered manually by selecting 'Get Token' or 'Force Refresh Token' under 'APP' section.